sdk/core/core-http/lib/restError.ts

```diff
@@ -14,13 +14,25 @@ export class RestError extends Error {
   request?: WebResource;
   response?: HttpOperationResponse;
   body?: any;
-  constructor(message: string, code?: string, statusCode?: number, request?: WebResource, response?: HttpOperationResponse, body?: any) {
+  parsedBody?: any;
+  parsedHeaders?: { [key: string]: any };
+  
+  constructor(
+    message: string,
+    code?: string,
+    statusCode?: number,
+    request?: WebResource,
+    response?: HttpOperationResponse,
+    body?: any,
```

Comment:
prefer `unknown` to `any`

Reply:
Changed for parsedHeaders since this is newly introduced in this PR. For the body, we will clean up in the future